### DISCUSSION TITLE
Support Nested(children) Sortable(drag-and-drop)

### DISCUSSION THREAD

**Discussion by EvodiaAut** [filamentphp/filament#3384] [Ideas]
Live JS example:
[sortablejs.github.io | Nested Sortables Example](https://sortablejs.github.io/Sortable/#nested)

Case example:
[demo.filamentphp.com | Shop > Brands](https://demo.filamentphp.com/shop/brands?tableSortColumn=sort&tableSortDirection=asc&isTableReordering=true)
-- Brand A1
--- Brand A1.1
--- Brand A1.2
---- Brand A1.2.1
---- Brand A1.2.2
-- Brand A2
-- Brand A3

Link for inspire:
https://github.com/lazychaser/laravel-nestedset

**Comment by pxlrbt** [comment]
Maybe this would be a good plug-in and could be adapted from Ryan's navigation plugin

**Comment by ksorsret** [comment]
How I can implement drag dog nested? Have solution for this

**Comment by cmayorgahilario** [comment]
Have solution for this?
